### ISSUE TITLE
French language support

### ISSUE THREAD

**Issue by zemadara** [allegro/tradukisto#25]
Hello, you do a really good job i just create this issue because I think i'm not authorized to create a branch, so if you could add some modifications for French language it would be very nice:
**in ValueConverters add** : 
`import static pl.allegro.finance.tradukisto.internal.Container.frenchContainer;`
`FRENCH_INTEGER(frenchContainer().getNumbersConverter())`

**in ValueConvertersTest add**:
```
def "should convert numbers in French"() {
        expect:
        FRENCH_INTEGER.asWords(1_234) == "un mille deux cent trente-quatre"
    }
```
if you want to be more precise, FRENCH_INTEGER.asWords(1_234) should return "mille deux cent trente-quatre" instead of "un mille deux cent trente-quatre", in french when we have one thousand (and other number) we don't write the one (like brazilian i think):
ex: 1000 = mille                                 (one thousand)
      1750 = mille sept cent cinquante (one thousand seven hundred fifty) 
      2000 = deux mille                        (two thousand)

**Comment by dorians** [comment]
You can fork this project and create a pull request from your fork.

**Comment by zemadara** [comment]
please merge and release

**Comment by wjur** [comment]
Thank you for your contribution, @zemadara. The fix is released in 1.6.0.
